index.bs

```diff
@@ -701,62 +707,103 @@ When this method is invoked, the user agent MUST execute the following algorithm
-                :   {{AuthenticatorAttestationResponse/attestationObject}}
-                ::  |attestationObject|
-            :   {{PublicKeyCredential/clientExtensionResults}}
-            ::  A new {{AuthenticationExtensions}} object containing the [=extension identifier=] → [=client extension output=]
-                entries created by running each extension's [=client extension processing=] algorithm to create the [=client
-                extension outputs=], for each [=client extension=] in {{AuthenticatorResponse/clientDataJSON}}.clientExtensions.
-
-        5. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
-            |authenticator| and [=set/remove=] it from |issuedRequests|.
-        6. Return |value| and terminate this algorithm.
-    </dd>
+        :   If the |adjustedTimeout| timer expires,
+        ::  [=set/For each=] |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on |authenticator|
+            and [=set/remove=] |authenticator| from |issuedRequests|.
+
+        :   If any |authenticator| returns a status indicating that the user cancelled the operation,
+        ::  1. [=set/Remove=] |authenticator| from |issuedRequests|.
+            2. [=set/For each=] remaining |authenticator| in |issuedRequests| invoke the [=authenticatorCancel=] operation on
+                |authenticator| and [=set/remove=] it from |issuedRequests|.
+
+        :   If any |authenticator| returns an error status,
+        ::  [=set/Remove=] |authenticator| from |issuedRequests|.
+
+        :   If any |authenticator| indicates success,
+        ::  1.  [=set/Remove=] |authenticator| from |issuedRequests|.
+
+            1.  Let |credentialCreationData| be a [=struct=] whose [=items=] are:
+
+                :   <code><dfn>attestationObjectResult</dfn></code>
+                ::  whose value is the bytes returned from the successful [=authenticatorMakeCredential=] operation.
+
+                    Note: this value is <code>attObj</code>, as defined in [[#generating-an-attestation-object]].
+
+                :   <code><dfn>clientDataJSONResult</dfn></code>
+                ::  whose value is the bytes of |clientDataJSON|.
+
+                :   <code><dfn>extensionOutputsMap</dfn></code>
+                ::  whose value is an [=ordered map=] with [=map/keys=] of type [=extension identifier=]
+                    and [=map/values=] of type [=client extension output=]. <code>[=extensionOutputsMap=]</code>'s
+                    [=map/entries=] are created by running each extension's [=client extension processing=] algorithm to create
+                    the [=client extension outputs=], for each [=client extension=] in
+                    <code>{{AuthenticatorResponse/clientDataJSON}}.clientExtensions</code>.
+
+            1.  Let |constructCredentialAlg| be an algorithm that takes a [=environment settings object/global object=]
```

Comment:
`takes a [=global object=]`, which should resolve to https://html.spec.whatwg.org/multipage/webappapis.html#global-object.

Reply:
ok.